data/data/aws/master/outputs.tf

```diff
@@ -2,3 +2,6 @@ output "ip_addresses" {
   value = aws_network_interface.master.*.private_ips
 }
 
+output "ipv6_addresses" {
```

Comment:
also similar question to https://github.com/openshift/installer/pull/2555#discussion_r354966038

Reply:
This one seems like a toss-up.  The master module does not take a `use_ipv6` input, which would be needed to figure out what to put in the `ip_addresses` output variable.  I could drop `ipv6_addresses`, but would add an input variable instead, so it doesn't seem much better.  I'll change it if you prefer it that way, though.